packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -192,13 +196,23 @@ export default class TabConsole extends Component {
               <span className='input-group-btn'>
                 <ActionButton
                   handler={this._openSshMore}
-                  tooltip={canSsh ? _('sshUserTooltip') : _('sshNeedClientTools')}
-                  disabled={!canSsh}
+                  tooltip={canSshOrRdp ? _('sshUserTooltip') : _('remoteNeedClientTools')}
+                  disabled={!canSshOrRdp}
                   icon='remote'
                 >
                   {_('sshUserLabel')}
                 </ActionButton>
               </span>
+              <span className='input-group-btn'>
+                <ActionButton
+                  handler={this._openRdp}
+                  tooltip={canSshOrRdp ? _('rdpRootTooltip') : _('remoteNeedClientTools')}
+                  disabled={!canSshOrRdp}
+                  icon='remote'
```

Comment:
I'd suggest [this icon](https://fontawesome.com/v4.7.0/icon/picture-o) or [this icon](https://fontawesome.com/v4.7.0/icon/desktop) to differentiate from the SSH buttons.